hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/FilterParams.java

```diff
@@ -107,17 +128,39 @@ public String getFilterBySearchText() {
      *
      * @return the flag indicating if tagging filter is used
      */
-    public Boolean getSelectTargetWithNoTag() {
+    public boolean isSelectTargetWithNoTag() {
         return selectTargetWithNoTag;
     }
 
     /**
      * Gets the tags that are used to filter for. The activation of this filter
-     * is done by {@link #setSelectTargetWithNoTag(Boolean)}.
+     * is done by {@link #isSelectTargetWithNoTag()}.
      *
-     * @return the tags that are used to filter for
+     * @return the tags that are used to filter for or an empty array if non are
+     *         present
      */
     public String[] getFilterByTagNames() {
         return filterByTagNames;
     }
+
+    public boolean isEmpty() {
+        return isEmpty(filterByDistributionId);
+    }
+
+    public boolean isEmpty(Long filterByDistributionId) {
+        return filterByStatus.isEmpty() //
+                || getFilterBySearchText().isPresent() //
```

Comment:
if we have a `filterBySearchTest`, does this mean that this `FilterParams` is empty?

Reply:
Add some description based on the logic of the method. But not sure if the words reflect the extact meaning of the method.

Reply:
I just tried to preserve the logic as it was implemented before (in different places).